Tests/GoogleMapsUtilsSwiftTests/unit/QuadTree/GQTPointQuadTreeTest.swift

```diff
@@ -0,0 +1,173 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+@testable import GoogleMapsUtils
+
+/// TO-DO: Rename the required linked classes once the linking is done and remove the objective c class.
+// A utility function to generate random numbers within a range
+func randd(min: Double, max: Double) -> Double {
+    let range = max - min
+    return min + range * Double(arc4random_uniform(1000)) / 1000.0
+}
+
+final class GQTPointQuadTreeTest: XCTestCase {
```

Comment:
This class and the methods within could use doc comments

Reply:
Done the changes, except test files.